opus/application/static_media/js/browse.js

```diff
@@ -1242,6 +1242,15 @@ var o_browse = {
 
     showDetail: function(e, opusId) {
         o_browse.hideMenus();
+
+        // if the item is selected as 'detail', add the icon in the corner
+        // hide all first... this will hide both on browse and cart
```

Comment:
Hide all what first? And why are you hiding anything? This comment is confusing. And what "icon" in the corner? Do you mean the word "detail"?